src/PrestaShopBundle/Form/Admin/Type/DateRangeType.php

```diff
@@ -130,10 +129,13 @@ public function adaptUnlimited(FormEvent $event): void
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
+            'placeholder' => 'YYYY-MM-DD',
             'date_format' => 'YYYY-MM-DD',
             'has_unlimited_checkbox' => false,
+            'default_end_value' => (new DateTime())->format('Y-m-d'),
```

Comment:
Why forcing a default end value here as well? This also potentially changes some wanted behaviour

Reply:
There is no change. Before, it was hardcoded in the 'attr' array above, see removed line `'data-default-value' => $now->format('Y-m-d'),`.

Now, it gets passed dynamicaly along with date_format and others. No change.